scripts/release.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -eu
+
+if [ -z "${1:-}" ]; then
+  echo "ERROR! Pass in the new version string"
+  exit 1
+fi
+
+NEW_VERSION="${1}"
```

Comment:
It's maybe a little confusing calling this NEW_VERSION.   If we are releasing v1.50 of Kiali, then the content in `staging` is for `v1.50`, the content in `current` is for `v1.49`.   I think perhaps this should be called CURRENT_VERSION.  Does that make any sense?

Reply:
Yeah, I actually questions that myself. I'll change to "CURRENT_VERSION" since it corresponds to the version of the 'current' branch.